_posts/2024-05-03-trino-snowflake-bloomberg-oss-win.md

```diff
@@ -0,0 +1,294 @@
+---
+layout: post
+title:  "Integrating Trino and Snowflake: An open source success story"
+author: Brian Olsen
+excerpt_separator: <!--more-->
+canonical_url: https://bitsondata.dev/trino-snowflake-bloomberg-oss-win
+---
+
+We’re seeing open source usher in a challenge to the economic model where the 
+success metric is increasing the commonwealth of economic capital. This 
+acceleration comes from playing positive-sum games with friends online and 
+avoiding limiting a community to a vision that only benefits a small number of 
+corporations or individuals. It’s hard to imagine how to embed such frameworks 
+within our current zero-sum winner-takes-all economic system. There’s certainly 
+no shortage of heated debates around how to construct a harmonious relationship 
+between the open source community and companies participating in them. Something
+we don’t talk about enough are the positive examples of when a coordinated 
+effort in open source sticks the landing and so many benefit from it.
+
+This post highlights the extraordinary contributions of 
+[Erik Anderson](https://www.linkedin.com/in/erikanderson/), 
+[Teng Yu](https://www.linkedin.com/in/tyu-fr/), 
+[Yuya Ebihara](https://www.linkedin.com/in/ebyhr/), and the broader 
+[Trino community](https://github.com/trinodb/trino) to finally contribute the 
+long-coveted 
+[Trino Snowflake Connector](https://trino.io/docs/current/connector/snowflake.html).
+
+It is both a success story and some wisdom for corporations using and providing
+services around open source should be involved in open source . This provides a
+blue print for those who want to contribute to open source the types of
+patterns to follow when contributing.
+
+## A common challenge in open source
+
+Despite my love for marketing and educating a community on tech (aka 
+[edutainment](https://en.wikipedia.org/wiki/Educational_entertainment)), it’s
+only the first part of the equation of what I aim to do as a developer advocate.
+Once developers  see some exciting video or tutorial, they ultimately land on
+the docs site, GitHub, StackOverflow, or some communication platform in the
+community. It's at this point that developers can easily lose the motivation if
+the the community and docs lacks proper getting started materials or worse, a
+silent community. This is how I categorize the developer experience (aka
+devex), which aims to improve both the user and contributor experiences in the
+developer community by
+[empower decision making by doing](https://en.wikipedia.org/wiki/Experiential_learning),
+[removing inefficiencies](https://trino.io/blog/2023/01/09/cleaning-up-the-trino-backlog),
+and as we'll cover here, exposing untapped opportunities. 
+
+Much like any open source project, maintainers on the Trino project struggle at
+communicating our lack of proper resources to build and test new features built
+on various proprietary software. For those less familiar, Trino is a federated
+query engine with
```

Comment:
I think we can probably leave out the brief explainer for what Trino is for the version of the blog going on the Trino website?

Reply:
It's going to be posted to my site and I want it to serve as a reference point for anyone I discuss with.